pkg/tasks/reset.go

```diff
@@ -29,6 +31,59 @@ import (
 	"k8s.io/apimachinery/pkg/util/wait"
 )
 
+// RemoveLBServices Deletes all the load balancers services from the cluster.
+func RemoveLBServices(s *state.State) error {
+	if !s.RemoveLBServices {
+		return nil
+	}
+	var lastErr error
+	s.Logger.Infoln("Deleting LoadBalancer Services...")
+	lastErr = clientutil.CleanupLBs(s.Context, s.Logger, s.DynamicClient)
+	if lastErr != nil {
+		s.Logger.Warn("Unable to delete services of type load balancer.")
+
+		return lastErr
+	}
+
+	lastErr = clientutil.WaitCleanupLbs(s.Context, s.Logger, s.DynamicClient)
+	if lastErr != nil {
+		s.Logger.Warn("Waiting for all load balancer services to be deleted...")
+
+		return lastErr
+	}
```

Comment:
```suggestion
	lastErr = clientutil.WaitCleanupLbs(s.Context, s.Logger, s.DynamicClient)
	if lastErr != nil {
		s.Logger.Warn("Waiting for all load balancer services to be deleted...")
	}
```

lastErr will be returned anyway